### ISSUE TITLE
Need ways to set language in addition to settings file

### ISSUE THREAD

**Issue by pieper** [Slicer/Slicer#6833]
Follow-on to discussion here: https://github.com/Slicer/Slicer/pull/6828

We could look at what other software does (installer option, runtime dialog, commandline flag, etc) and see what's most useful for our users.

**Comment by lassoan** [comment]
I've added a command-line option to set the language like this: `Slicer.exe --language hu` - see this branch: https://github.com/lassoan/Slicer/commits/set-language-via-commandline 

However, in itself it is not useful yet, because currently we don't bundle any language packs in the installation package. Also, it may be better to just change the language without starting the Slicer application and then if the user wants to use the application in the new language then he can start the application after the language is set. This would not need any Slicer code change but could be achieved by this command:

```
Slicer.exe -c "import slicer; slicer.app.userSettings().setValue('language', 'hu'); slicer.app.userSettings().setValue('Internationalization/Enabled', True)" --no-splash
```

This command could be used in a shell script file that collects/installs translation files and configures Slicer for that language.

This command could also be used during installation. The installer could show a language selector popup and write the result into the application settings. Language packs could be bundled with the installer or the application could automatically download the necessary language packs at startup.

Since the above Python command can be already used, I don't think any development is needed for Slicer-5.3.

**Comment by pieper** [comment]
Yes, doing this as a python script makes sense - no need to complicate the launcher.  For running tests or generating tutorials it would also be nice to have a script that pulls down everything (or just selected languages) from weblate.

I like the idea of offering to install language as part of the installer.
